samples/nrf9160/modem_shell/src/gnss/gnss_interface.c

```diff
@@ -527,21 +670,20 @@ static int inject_agps_data(void *agps,
 
 	if (err) {
 		mosh_error(
-			"GNSS: Failed to send AGPS data, type: %s (err: %d)",
+			"GNSS: Failed to send A-GPS data, type: %s (err: %d)",
 			type_string,
 			errno);
```

Comment:
This printing as well as next one fits into one line

Reply:
This one doesn't fit on one line. I'll fit this one on two lines and the following one on one line.

Reply:
I must have had 4 character tab when testing so true.